### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by SHCHERBANV**

### CODE REVIEW

**Comment by SHCHERBANV** [comment]
for review

File: app/main.py [hunk 1]
```diff
@@ -1,17 +1,8 @@
...
 
-    :param phrase: phrase to count in it
-    :param letter: letter to find occurrences of it
-    :return: count occurrences of letter in phrase
-    """
-    # write your code here
+def count_occurrences(phrase: str, letter: str) -> int:
+    phrase = phrase.lower()
+    letter = letter.lower()
+    count = Counter(phrase)[letter]
```
Comment (NazarSheremet) [inline app/main.py:7]: No need to import `Counter`. You can use the built-in function `count()`

**Review by NazarSheremet — CHANGES_REQUESTED** [review]
Need changes

**Comment by SHCHERBANV** [comment]
def count_occurrences(phrase: str, letter: str) -> int:
    phrase = phrase.lower()
    count = phrase.count(letter.lower())
    return count

**Pull request closed without merging**